crates/miden-lib/asm/account_components/no_auth.masm

```diff
@@ -0,0 +1,5 @@
+use.miden::account
+
+export.auth__basic
+    push.1 exec.account::incr_nonce
+end
```

Comment:
A couple of comments:

- After [#1608](https://github.com/0xMiden/miden-base/pull/1608), `incr_nonce` does not take any parameters (it always increments the nonce by 1).
- I would make the logic a bit more complicated. We should check if the account's state changed, and increment the nonce only if it did. If the state didn't change, no need to increment the nonce.
- I would maybe use a different name instead of `auth__basic`. Maybe `auth__none` (though I don't love it either).

Reply:
> After [#1608](https://github.com/0xMiden/miden-base/pull/1608), `incr_nonce` does not take any parameters (it always increments the nonce by 1).

Just to point this out: #1608 was merged, but against `next` and since this targets `main` I think the `push.1` is still correct.